v22.2/security-reference/scram-authentication.md

```diff
@@ -67,10 +67,6 @@ This section details how to use SCRAM authentication rather than plaintext passw
 1. [Enable SCRAM-SHA-256 password encryption](#enabling-scram-sha-256-authentication-for-new-users-roles)
 2. [Migrate pre-existing users](#migrating-existing-users-roles-to-scram-sha-256-authentication)
 
-{{site.data.alerts.callout_info}}
-SCRAM authentication can only be used after the upgrade to v22.1 has been finalized.
-{{site.data.alerts.end}}
-
```

Comment:
@MichaelTrestman Do we need a similar but revised note for the v22.2 docs? If a customer were on an upgrade path from v21.2, would't this still be true, but need slight rewording, e.g.

> If you are running a version of CockroachDB prior to v22.1, SCRAM authentication can only be used after an upgrade to v22.1 has been finalized.

Reply:
Thanks, Andrew. BTW, just fixed my blockquote - I meant for the last version number there to be "v21.2".

Reply:
Yep, since upgrading to v22.1 is a required step to upgrade to v22.2 and above, we figured that note was no longer necessary.

Reply:
Yup, I concur that the note is no longer needed!